src/pages/Swap/index.tsx

```diff
@@ -314,10 +314,20 @@ export function Swap({
     [independentField, parsedAmount, showWrap, trade]
   )
 
-  const fiatValueInput = useUSDPrice(parsedAmounts[Field.INPUT], currencies[Field.INPUT] ?? undefined)
-  const fiatValueOutput = useUSDPrice(parsedAmounts[Field.OUTPUT], currencies[Field.OUTPUT] ?? undefined)
-  const showFiatValueInput = Boolean(parsedAmounts[Field.INPUT])
-  const showFiatValueOutput = Boolean(parsedAmounts[Field.OUTPUT])
+  const fiatValueInput = useUSDPrice(
+    parsedAmounts[Field.INPUT] ??
+      (currencies[Field.INPUT]
+        ? CurrencyAmount.fromRawAmount(currencies[Field.INPUT], JSBI.BigInt(10 ** currencies[Field.INPUT].decimals))
+        : undefined),
+    currencies[Field.INPUT] ?? undefined
```

Comment:
`?? undefined` i think that can just be removed

Reply:
unless currencies can return `null` and it only wants an undefined type there which would be odd